src/Package/TestApp/Files/VariableGrid02.tree

```diff
@@ -520,7 +520,7 @@
                                       "Properties": [
                                         {
                                           "Name": "Text",
-                                          "Value": "V1"
+                                          "Value": "[,1]"
```

Comment:
RStudio shows V1 that's why it was changed back then. I frankly don't have background why we were different